qa/L0_storage_S3/test.sh

```diff
@@ -41,7 +41,7 @@ CLIENT_LOG_BASE="./client"
 INFER_TEST=infer_test.py
 
 # S3 credentials are necessary for this test. Pass via ENV variables
-aws configure set default.region us-west-1 && \
+aws configure set default.region $AWS_DEFAULT_REGION && \
```

Comment:
If aws is always configured this way then how are you testing with just env vars? It seems there are 3 cases. With just envvars, with just "aws configure" and with both, where you set the lower priority method to some bogus values to make sure that it is not used.

Reply:
Fixed

Reply:
I still don't see how this is working. You are unconditionally setting this outside the loop. Shouldn't you be doing this in the loop as the last tested case?